src/java.base/share/man/java.md

```diff
@@ -4011,34 +4011,34 @@ application is executed. One example is Class Data Sharing (CDS), as described a
 that parses classes ahead of time. AOT optimizations can improve the start-up and
 warm-up performance of Java applications.
 
-The Ahead-of-Time Cache (AOTCache) is a container introduced in JDK 24 for
-storing artifacts produced by AOT optimizations. The AOTCache currently contains
-Java classes and heap objects. The plans is to include other types of artifacts,
-such as execution profiles and compiled methods, in future JDK releases.
+The Ahead-of-Time Cache (AOT Cache) is a container introduced in JDK 24 for
+storing artifacts produced by AOT optimizations. The AOT cache currently contains
+Java classes and heap objects. In future JDK releases, the AOT cache may contain additional
+artifacts, such as execution profiles and compiled methods.
 
-An AOTCache is specific to a combination of the following:
+An AOT cache is specific to a combination of the following:
 
 -   A particular application (as expressed by `-classpath`, `-jar`, or `--module-path`.)
 -   A particular JDK release.
 -   A particular OS and CPU architecture.
 
-If any of the above changes, you must recreate the AOTCache.
+If any of the above changes, you must recreate the AOT cache.
```

Comment:
Maybe add "The main class may be changed."

Reply:
You can change the main class or parameters to the application and will still be able to use the cache, although if you execute new code that was cover during the training run then you will have less benefit from the cache.